client/analytics/report/categories/index.js

```diff
@@ -55,6 +62,10 @@ export default class CategoriesReport extends Component {
 				: 'category';
 		}
 
+		filters[ 0 ].filters.filter(
+			( item ) => item.value === 'compare-categories'
+		)[ 0 ].settings.onClick = addCesSurveyTrackForAnalytics;
```

Comment:
Filters have filters? Mildly confusing (and pre-existing). 😐 Anyway...

I think using `find()` instead of `filter()` would be a bit clearer, as you are always using the first item found by the `filter()`.

Reply:
@mattsherman Thank you 👍 I agree that `find()` is better for this case. Updated in https://github.com/woocommerce/woocommerce-admin/pull/5649/commits/291516cf5f50e30defd6fd6e20c2b40fd00d1203